src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -46,8 +46,14 @@ ior:
     # This test doesn't work if the IOR runs too fast. In that case, increase
     # block_size and/or decrease transfer_size. To decrease transfer_size, divide it
     # by 2, 4, 8, etc.
-    transfer_size: 256K
+    transfer_size: 256K  # CI
+    # transfer_size: 1G  # Aurora
     block_size: 100G
     sw_deadline: 60
-    dfs_oclass: SX
-    dfs_dir_oclass: SX
+ior_wo_rf:
+  <<: *ior_base
+  dfs_oclass: SX
+ior_with_ec:
+  <<: *ior_base
+  dfs_oclass: EC_2P2GX  # CI
+  # dfs_oclass: EC_16P2GX  # Aurora
```

Comment:
FYI, due to how avocado parses this syntax, your extra yaml would have to look like this
```
ior_wo_rf:
  <<:
    dfs_oclass: OVERRIDE
    transfer_size: OVERRIDE
ior_with_ec:
  <<:
    dfs_oclass: OVERRIDE
    transfer_size: OVERRIDE
```

Reply:
I'm not sure what you mean. I'm using the default extra yaml and getting the expected IOR command.
```
/scratchbox/daos/install-latest-ecb/ior/bin/ior -a DFS -b 100G -k -v -w
-W -D 60 -o /test_file_2 -t 10G --dfs.chunk_size 1048576 --dfs.cont TestContainer_1 --dfs.dir_oclass SX --dfs.oclass EC_16P2GX --dfs.pool TestPool_1
```

Reply:
You mean `cxi.yaml`? How do you tell the test to use `EC_16P2GX` instead of `EC_2P2GX`?

Reply:
Just uncomment line 59 (`EC_16P2GX`) and comment line 58 (`EC_2P2GX`). I don't have to touch `cxi.yaml`.

Reply:
I think it's better to use an extra yaml like these
https://github.com/daos-stack/aurora-tools/tree/master/pbs_scripts/extra_yaml
Otherwise we have to write documentation on how/what to change and have to update the file every time we run.
And if we're running out of a shared source build, we can't just update the installed config without messing up someone else

Reply:
Okay. I'll create an extra yaml for this test. Thanks.